fastlane/Fastfile

```diff
@@ -204,6 +214,32 @@ lane :tag_current_branch do |options|
   push_git_tags(tag: version_number)
 end
 
+private_lane :trigger_action do |options|
```

Comment:
We can move this to the common fastlane plugin, but since I had already added it here, I figured we can have it here until its moved over there, which takes time due to unit tests and such

Reply:
Hmm yeah, I think this would ideally be there, since it seems like something we might want to reuse... But I'm ok with this for now.